Chtotiskazal/Chotiskazal.Bot/ChatFlows/LastTranslationHandler.cs

```diff
@@ -14,7 +14,7 @@ public class LastTranslationHandler {
         private ChatRoom Chat { get; }
         private readonly AddWordService _addWordService;
         private readonly IReadOnlyList<Translation> _translations;
-        private readonly LongDataForButtonService _longDataForButtonService;
+        private readonly CallbackDataForButtonService _callbackDataForButtonService;

```

Comment:
Rename to ButtonCallbackDataService, as CallBackData For ButtonService is confusing